qcost

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+import pandas
+import argparse
+import re
+import sys
+
+from datetime import datetime, time, timedelta
+from io import StringIO
+from math import log
+
+queue_info = None
+
+unit_base = { 'B' : 1024, 'SU' : 1000 }
+
+def extract_num_unit(s):
+    # Match a number (possibly floating point 100.00 style) and a unit
+    try:
+        size, unit = re.findall(r'(\d+.\d+|\d+)\s*(\D*)$',s)[0]
+    except:
+        print('Failed to match size string: ',s)
+        sys.exit()
+    return float(size), unit
+
+def pretty_size(n,pow=0,b=1024,u='B',pre=['']+[p for p in'KMGTPEZY']):
```

Comment:
for info no need to change - dask has dask.utils.format_bytes and dask.utils.parse_bytes which can be handy for this.

Reply:
Good to know. Tah.